microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/activation/ActivationPropertiesUnsetTest.java

```diff
@@ -0,0 +1,80 @@
+ */
+@RunWith(Arquillian.class)
+public class ActivationPropertiesUnsetTest {
+
+    private static final String NON_JWT_AUTH_ACTIVATION_PROPERTY_DEPLOYMENT_NAME = "PROPERTIES_UNSET_DEPLOYMENT";
+    private static final String UNSET_AUTH_ACTIVATION_PROPERTY_DEPLOYMENT_NAME = "OTHER_VALUE_DEPLOYMENT";
+
+    @Deployment(name = NON_JWT_AUTH_ACTIVATION_PROPERTY_DEPLOYMENT_NAME, managed = false)
+    public static Archive<?> createDeploymentNoValue() {
+        return ShrinkWrap
+                .create(WebArchive.class,
+                        ActivationPropertiesUnsetTest.class.getSimpleName() + NON_JWT_AUTH_ACTIVATION_PROPERTY_DEPLOYMENT_NAME
+                                + ".war")
+                .addClass(SecuredJaxRsEndpoint.class)
+                .addClass(NonJwtJaxRsTestApplication.class)
+                .addAsManifestResource(
+                        ActivationPropertiesUnsetTest.class.getClassLoader().getResource("mp-config-basic.properties"),
+                        "microprofile-config.properties")
+                .addAsManifestResource(
+                        ActivationPropertiesUnsetTest.class.getClassLoader().getResource("pki/key.public.pem"),
+                        "key.public.pem");
+    }
+
+    @Deployment(name = UNSET_AUTH_ACTIVATION_PROPERTY_DEPLOYMENT_NAME, managed = false)
+    public static Archive<?> createDeploymentOtherValue() {
+        return ShrinkWrap
+                .create(WebArchive.class,
+                        ActivationPropertiesUnsetTest.class.getSimpleName() + UNSET_AUTH_ACTIVATION_PROPERTY_DEPLOYMENT_NAME
+                                + ".war")
+                .addClass(SecuredJaxRsEndpoint.class)
+                .addClass(NonJwtJaxRsTestApplication.class)
+                .setWebXML(ActivationPropertiesUnsetTest.class.getClassLoader().getResource("basic-auth-web.xml"))
+                .addAsManifestResource(
+                        ActivationPropertiesUnsetTest.class.getClassLoader().getResource("mp-config-basic.properties"),
+                        "microprofile-config.properties")
+                .addAsManifestResource(
+                        ActivationPropertiesUnsetTest.class.getClassLoader().getResource("pki/key.public.pem"),
+                        "key.public.pem");
+    }
+
+    /**
+     * @tpTestDetails Test that subsystem is not activated when there is not {@code MP-JWT} set as an
+     *                {@code auth-method} anywhere (both web.xml and {@code @LoginConfig} annotation).
+     * @tpPassCrit Deployment fails because classes required by deployment (MP-JWT API) are not loaded due to the fact
+     *             that the subsystem was not activated.
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test(expected = org.jboss.arquillian.container.spi.client.container.DeploymentException.class)
+    public void testDeploymentFailsWithUnsetAuth(@ArquillianResource Deployer deployer) {
+        deployer.deploy(NON_JWT_AUTH_ACTIVATION_PROPERTY_DEPLOYMENT_NAME);
```

Comment:
Yes definitely it seems to me there is something wrong with those constants' name. Please see my comment on their declaration.

Reply:
Done